challenge.md

```diff
@@ -4,21 +4,21 @@
 
 Write your answers inside this file, where it's indicated by the comments.
 
-1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that? git clone
```

Comment:
`git clone` alone will not work; you also need to specify the URL.

e.g.
`git clone {SSH URL}`